x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ClusterComputeResponse.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.plugin;
+
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.compute.operator.DriverProfile;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+
+final class ClusterComputeResponse extends ActionResponse {
+    private final Collection<DriverProfile> profiles;
```

Comment:
I wonder if we should combine this with `DataNodeResponse`. I couldn't tell you the difference between `ActionResponse` and `TransportResponse`, but that looks like the only differentce.

Reply:
Yes, I pushed https://github.com/elastic/elasticsearch/pull/101640/commits/5c04c0dc0483b4e3b06ca284075b2e0ab648aba3 to share the responses.